app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,32 @@
+class RelationshipsController < ApplicationController
+  before_action :signed_in_user?, only: [:create, :destroy]
```

Comment:
good idea to put this in the before action. I'm going to implement something similar.